src/schemas/execute.yaml

```diff
@@ -265,10 +265,21 @@ CallResultFailure:
       $ref: '#/components/schemas/uint64'
     error:
       oneOf:
-        - code: -32000
-          message: 'Execution reverted'
-        - code: -32015
-          message: 'VM execution error'
+      - type: object
+        properties:
+          code:
+            const: -32000
+          message:
+            type: string
+            pattern: "^execution reverted.*"
+        required: [code, message]
+      - type: object
+        properties:
+          code:
+            const: -32015
+          message:
+            type: string
+            pattern: "^vm execution error.*"
```

Comment:
Does case matter here?  I haven't tested, but I thought that these were both capital first letter casing?